root/dataframe/test_trainCache.C

```diff
@@ -1,3 +1,5 @@
+R__LOAD_LIBRARY(libROOTDataFrame);
```

Comment:
why is this required? or: why was it not required before?

Reply:
I don't know, but without that it fails on Windows...

Reply:
alright, can you please enclose it in an `#ifdef WINDOWS`, if nothing else for documentation purposes?

Reply:
(rationale: we want to keep testing that autoloading works on other platforms)

Reply:
Fair enough, will do

Reply:
Done